modules/features/podcasts/src/main/java/au/com/shiftyjelly/pocketcasts/podcasts/view/components/ratings/GiveRatingScreen.kt

```diff
@@ -0,0 +1,198 @@
+package au.com.shiftyjelly.pocketcasts.podcasts.view.components.ratings
+
+import android.widget.Toast
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.size
+import androidx.compose.material.Icon
+import androidx.compose.material.MaterialTheme
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.filled.Star
+import androidx.compose.material.icons.filled.StarBorder
+import androidx.compose.material.icons.filled.StarHalf
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.vector.ImageVector
+import androidx.compose.ui.platform.LocalContext
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.tooling.preview.PreviewParameter
+import androidx.compose.ui.unit.dp
+import au.com.shiftyjelly.pocketcasts.compose.AppThemeWithBackground
+import au.com.shiftyjelly.pocketcasts.compose.bars.NavigationButton
+import au.com.shiftyjelly.pocketcasts.compose.bars.NavigationIconButton
+import au.com.shiftyjelly.pocketcasts.compose.buttons.RowButton
+import au.com.shiftyjelly.pocketcasts.compose.components.PodcastCover
+import au.com.shiftyjelly.pocketcasts.compose.components.TextH30
+import au.com.shiftyjelly.pocketcasts.compose.preview.ThemePreviewParameterProvider
+import au.com.shiftyjelly.pocketcasts.compose.theme
+import au.com.shiftyjelly.pocketcasts.podcasts.viewmodel.GiveRatingViewModel
+import au.com.shiftyjelly.pocketcasts.podcasts.viewmodel.GiveRatingViewModel.State.Loaded.Stars
+import au.com.shiftyjelly.pocketcasts.ui.theme.Theme
+import au.com.shiftyjelly.pocketcasts.localization.R as LR
+
+@Composable
+fun GiveRatingScreen(
+    state: GiveRatingViewModel.State.Loaded,
+    setStars: (Stars) -> Unit,
+    submitRating: () -> Unit,
+    onDismiss: () -> Unit,
+) {
+
+    val context = LocalContext.current
+    Content(
+        state = state,
+        setStars = { setStars(it) },
+        submitRating = {
+            Toast.makeText(context, "TODO: Submit rating", Toast.LENGTH_LONG).show()
+            submitRating()
+        },
+        onDismiss = onDismiss,
+    )
+}
+
+@Composable
+private fun Content(
+    state: GiveRatingViewModel.State.Loaded,
+    setStars: (Stars) -> Unit,
+    submitRating: () -> Unit,
+    onDismiss: () -> Unit,
+) {
+    Column(
+        horizontalAlignment = Alignment.CenterHorizontally,
+        modifier = Modifier.fillMaxSize()
+    ) {
```

Comment:
Can we make it scrollable? I wasn't able to scroll the screen in landscape mode.

Reply:
Thanks for catching that. Added in https://github.com/Automattic/pocket-casts-android/pull/1451/commits/ac82dc4015f7971d0bcb7f72b5d9d9d917458449